src/main/kotlin/com/example/travelermileageservice/application/EventService.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.travelermileageservice.application
+
+import com.example.travelermileageservice.domain.Point
+import com.example.travelermileageservice.domain.Review
+import com.example.travelermileageservice.model.dto.EventDto
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.UUID
+
+@Service
+class EventService(
+    private val reviewService: ReviewService,
+    private val pointService: PointService
+) {
+    @Transactional
+    fun addSave(eventDto: EventDto) {
+        validation(eventDto.reviewId)
+        reviewService.save(Review.of(eventDto))
+        pointService.save(Point.of(eventDto, 1))
+    }
+
+    private fun validation(reviewId: UUID) {
```

Comment:
함수는 동사로 표현하는게 일박적이고 직관적입니다.
```suggestion
    private fun validate(reviewId: UUID) {
```